### ISSUE TITLE
[Bug]: can't start docker container in ubuntu

### ISSUE THREAD

**Issue by devrk96** [appsmithorg/appsmith#1946]
## Description
After the recent changes, I tried restarting wildcard-nginx but its status is showing `Exited` only.

### Steps to reproduce the behaviour:
1. I have removed the wildcard-nginx container first.
2. Then used `./start-https.sh https://release.app.appsmith.com` command.
3. Then I used `docker ps -a` to find the status of container but it was showing `Exited`.

### Important Details
 - Version: [20.04.1]
 - OS: [Ubuntu]
 - Browser [Chrome]

**Comment by hetunandu** [comment]
Please add logs here @devrk96

**Comment by devrk96** [comment]
@ofpiyush Please find the logs here

`
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: Getting the checksum of /etc/nginx/conf.d/default.conf
10-listen-on-ipv6-by-default.sh: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
2020/11/27 08:23:04 [emerg] 1#1: host not found in upstream "host.docker.internal" in /etc/nginx/conf.d/app.conf:29
nginx: [emerg] host not found in upstream "host.docker.internal" in /etc/nginx/conf.d/app.conf:29
`

**Comment by ofpiyush** [comment]
For the simplest quickfix in `app/client/start-https.sh` change line 67 from 

```
proc_version="$(cat /proc/version)"
```

to 

```
proc_version="microsoft"
```


I'll test why it worked on windows the last time and push a proper fix in a few minutes.

**Comment by mohanarpit** [comment]
@ofpiyush The user's problem is on Ubuntu. Are you sure `proc_version="microsoft"` will fix the issue?

**Comment by ofpiyush** [comment]
@mohanarpit yes the fix in #1891 was wrong, it replaced a negative if with a positive one, essentially making the script do the exact opposite thing for *both* linux and WSL.

My nginx in WSL was running which kept working, made me believe that the new script was working.
In my rush to hotfix the script to work for mac users, I didn't notice that my docker had crashed too. 🙈
